### PR TITLE
#264 add link to hipchat message

### PR DESCRIPTION
**Pull request by drummerwolli**
#264 add link to hipchat message

### CODE REVIEW

**Comment by drummerwolli** [comment]
text of the link is up for discussion, i did not come up with something better for now 🐳

File: zmon_worker_monitor/zmon_worker/notifications/hipchat.py [hunk 1]
```diff
@@ -19,8 +21,15 @@ def notify(cls, alert, *args, **kwargs):
 
         color = 'green' if alert and not alert.get('is_alert') else kwargs.get('color', 'red')
 
+        zmon_host = kwargs.get('zmon_host', cls._config.get('zmon.host'))
```
Comment (vetinari) [inline zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: I'd move this (excluding `message_text = ...` of course) inside the `if kwargs.get('link', False):`

File: zmon_worker_monitor/zmon_worker/notifications/hipchat.py [hunk 2]
```diff
@@ -19,8 +21,15 @@ def notify(cls, alert, *args, **kwargs):
...
+        alert_id = alert['alert_def']['id']
+        alert_url = urlparse.urljoin(zmon_host, '/#/alert-details/{}'.format(alert_id)) if zmon_host else ''
+        message_text = cls._get_subject(alert, custom_message=kwargs.get('message'))
+        if kwargs.get('link', False):
+            message_text += ' -- <a href="{}">GO TO ALERT</a>'.format(alert_url)
```
Comment (vetinari) [inline zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: probably lower case, this is "too loud" IMO

**Comment by vetinari** [comment]
Please prepare a PR for https://github.com/zalando-zmon/zmon-docs/blob/master/docs/user/notifications/hipchat.rst also to add the new parameters there

File: zmon_worker_monitor/zmon_worker/notifications/hipchat.py (hunk 2 above)
Comment (vetinari) [reply zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: and add a `link_text` in kwargs which defaults to "go to alert"

Comment (vetinari) [reply zmon_worker_monitor/zmon_worker/notifications/hipchat.py]: @elauria should the `<a href=` also have a `target="_blank"` (or what ever is needed to open in a new tab)?

**Comment by drummerwolli** [comment]
@vetinari all adjusted, please check again. thx for the feedback!

**Comment by vetinari** [comment]
:+1:

**Comment by lmineiro** [comment]
👍

**Pull request merged**